src/index.js

```diff
@@ -0,0 +1,16 @@
+/*
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
+*/
+import loaderUtils from 'loader-utils';
+
+export default function (source) {
+  let value = typeof source === 'string' ? JSON.parse(source) : source;
+  const options = loaderUtils.getOptions(this) || {};
+  value = JSON.stringify(value)
+    .replace(/\u2028/g, '\\u2028')
+    .replace(/\u2029/g, '\\u2029');
+  value = options.stringify ? `'${value}'` : value;
+  const module = this.version && this.version >= 2 ? `export default ${value};` : `module.exports = ${value};`;
```

Comment:
webpack 2 has JSON support in core, so better display a warning to about that && remove `json-loader` from `webpack.config.js`

 ```js
if (this.version && this.version < 2)
   const module = `module.exports = ${value};`;
   return cb(null, module)
}

this.emitWarning(`message`)
return cb(null, value) // not 100% about that :)
```